buf.go

```diff
@@ -77,9 +77,9 @@ func fbufalloc() []rune {
 // TODO(flux): This is another design constraint of Buffer - we want to efficiently
 // present contiguous segments of bytes, possibly by merging/flattening our tree
 // when a view is requested. This should be a rare operation...
-func (b *Buffer) View(q0, q1 int) []byte {
+func (b *Buffer) View(q0, q1 int) []rune {
```

Comment:
Non-actionable comment: I note in passing that I plan on switching Edwood to be largely `[]byte` based instead of using runes.